test/unit/server/handlers/consents.test.ts

```diff
@@ -163,7 +172,7 @@ describe('server/handlers/consents', (): void => {
       expect(response).toBe(202)
       jest.runAllImmediates()
 
-      expect(setImmediate).rejects.toThrowError()
+      // expect(setImmediate).rejects.toThrowError()
```

Comment:
why is this code commented?

Reply:
ah sorry that comment should be removed - setImmediate itself will not be throwing an error as the try catch exists within the callback function within it